documentation/docs/10-getting-started/20-creating-a-project.md

```diff
@@ -11,7 +11,7 @@ npm install
 npm run dev
 ```
 
-The first command will scaffold a new project in the `my-app` directory asking you if you'd like to set up some basic tooling such as TypeScript. See the FAQ for [pointers on setting up additional tooling](../faq#integrations). The subsequent commands will then install its dependencies and start a server on [localhost:5173](http://localhost:5173).
+The first command will scaffold a new project in the `my-app` directory asking you if you'd like to set up some basic tooling such as TypeScript. See the FAQ for [pointers on setting up additional tooling](/docs/frequently-asked-questions#integrations). The subsequent commands will then install its dependencies and start a server on [localhost:5173](http://localhost:5173).
```

Comment:
we made these relative to support serving the site on things like IPFS. not that we're actually doing that at the moment, but it's been semi-useful to be able to test support for things like that with this site
```suggestion
The first command will scaffold a new project in the `my-app` directory asking you if you'd like to set up some basic tooling such as TypeScript. See the FAQ for [pointers on setting up additional tooling](../frequently-asked-questions#integrations). The subsequent commands will then install its dependencies and start a server on [localhost:5173](http://localhost:5173).
```